numpy/core/multiarray.py

```diff
@@ -1154,28 +1154,39 @@ def packbits(myarray, axis=None):
     and 32 = 0010 0000.
 
     """
-    return (myarray,)
+    return (a,)
 
 
 @array_function_from_c_func_and_dispatcher(_multiarray_umath.unpackbits)
-def unpackbits(myarray, axis=None):
+def unpackbits(a, axis=None, count=None):
     """
-    unpackbits(myarray, axis=None)
+    unpackbits(a, axis=None, count=None)
 
     Unpacks elements of a uint8 array into a binary-valued output array.
 
-    Each element of `myarray` represents a bit-field that should be unpacked
-    into a binary-valued output array. The shape of the output array is either
-    1-D (if `axis` is None) or the same shape as the input array with unpacking
-    done along the axis specified.
+    Each element of `a` represents a bit-field that should be unpacked
+    into a binary-valued output array. The shape of the output array is
+    either 1-D (if `axis` is `None`) or the same shape as the input
+    array with unpacking done along the axis specified.
 
     Parameters
     ----------
-    myarray : ndarray, uint8 type
+    a : ndarray, uint8 type
        Input array.
     axis : int, optional
         The dimension over which bit-unpacking is done.
-        ``None`` implies unpacking the flattened array.
+        `None` implies unpacking the flattened array.
+    count : int or None, optional
+        The number of elements to unpack along `axis`, provided as a way
+        of undoing the effect of packing a size that is not a multiple
+        of eight. A non-negative number means to only unpack `count`
+        bits. A negative number means to trim off that many bits from
+        the end. `None` means to unpack the entire array (the default).
+        Counts larger than the available number of bits will add zero
+        padding to the output. Negative counts must not exceed the
+        available number of bits.
+
+        .. versionadded:: 1.17.0
```

Comment:
Single quotes are a shortcut for :py:`value` and are meant to be linked to python documentation. When creating the link fails, the text is rendered in italics. Since the intention here is for code formatting, they should all be two quotes. You can see the difference in the way the default axis argument `None` used to be rendered [here](https://docs.scipy.org/doc/numpy-1.15.4/reference/generated/numpy.unpackbits.html?highlight=unpackbits#numpy.unpackbits) vs this PR [here](https://5920-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/generated/numpy.unpackbits.html?highlight=unpackbits#numpy.unpackbits)

Reply:
@mattip. Most of the docs I see have single quotes. I prefer the aesthetic of the italicized version to the code formatted one, and technically None is a python object. However, I don't care much one way or the other, so will fix as you request ASAP.

Reply:
It is strange that with single quotes, it wouldn't link to the python documentation of `None` already. Indeed, that seems a bug...

Reply:
I think I  slightly prefer the double backticks for None (there is no point in linking it to the python docs anyway).

https://docs.scipy.org/doc/numpy/docs/howto_document.html has some examples using single ticks for the kwargs, and actually nothing for `None`. Could be nice to just decide for once, not sure I care though.

Reply:
I'll grep and count both versions in the existing code. My intuition tells me that single ticks dominate, but I'm not sure. Majority rules? :)

Reply:
`None` is just an example here. There are other single-ticked words in the documentation change: a, count, index,

Reply:
Yeah, although for a, count and axis at least, they are arguments, and the howto document examples use single backticks for those. However, while I would guess a lot of people somewhat use that document, I doubt it is applied consistently with respect to these examples in any case.

Reply:
Let's call it a niggling nit and ignore it, is this ready for merge otherwise?

Reply:
There were 145 occurrences of None with single quotes and 147 occurrences with double quotes until this PR came along. I will change back to double. Sorry for the holdup.

Reply:
@mattip If you want me to go through and normalize all the other occurences, I can do that fairly quickly, since I have the full list and a script that can do it right in front of me.